new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -1080,6 +1105,31 @@ public static void createSecretWithUsernamePassword(String secretName,
     assertTrue(secretCreated, String.format("create secret failed for %s", secretName));
   }
 
+  /**
+   * Create a RCU secret with username, password and sys_username, sys_password in the specified namespace.
+   *
+   * @param secretName secret name to create
+   * @param namespace namespace in which the secret will be created
+   * @param username username in the secret
+   * @param password passowrd in the secret
+   */
+  public static void createRcuSecretWithUsernamePassword(String secretName, String namespace,
+      String username, String password, String sysUsername, String sysPassword) {
```

Comment:
missing params sysUsername and sysPassword in javadoc

Reply:
Fixed